src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java

```diff
@@ -51,5 +53,6 @@ public Envelope(
         this.documents = documents;
         this.ocrData = ocrData;
         this.ocrValidationWarnings = ocrValidationWarnings;
+        this.formType = formType;
```

Comment:
minor thing, but let's me consisted and keep the order of arguments

Reply:
fixed